src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Database/DatabaseInfo.cs

```diff
@@ -49,4 +51,20 @@ public class DatabaseScopedConfigurationsInfo
         public string ValueForPrimary { get; set; }
         public string ValueForSecondary { get; set; }
     }
+
+    public class DatabaseFile
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string Type { get; set; }
+        public string Path { get; set; }
+        public string FileGroup { get; set; }
+        public string FileNameWithExtension { get; set; }
+        public double SizeInMb { get; set; }
+        public bool IsAutoGrowthEnabled { get; set; }
+        public double AutoFileGrowth { get; set; }
+        public string AutoFileGrowthType { get; set; }
+        public double MaxSizeLimit { get; set; }
```

Comment:
units?

Reply:
Sorry! didn't get this comment, anything wrong here?

Reply:
This doesn't say what units this should be...is it KB? MB? Something else? Any vars that store numeric values should generically have their units appended to the name for easier understanding.